### ISSUE TITLE
Custom build failures for Yarn Berry

### ISSUE THREAD

**Issue by samhh** [mattiasw/ExifReader#260]
Hey 🙂  I've found that custom builds don't work for Yarn Berry (i.e. Yarn v2+). Classic Yarn v1 appears to work. Here's a repro: https://git.sr.ht/~samhh/exifreader-repro/tree

```console
$ yarn install
<fails>

$ cat /path/to/build.log
/bin/sh: webpack: command not found
node:child_process:965
    throw err;
    ^

Error: Command failed: webpack
    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at Object.<anonymous> (/Users/sam/Dev/exifreader-repro/node_modules/exifreader/bin/build.js:12:5)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:86:12)
    at node:internal/main/run_main_module:23:47 {
  status: 127,
  signal: null,
  output: [ null, null, null ],
  pid: 26282,
  stdout: null,
  stderr: null
}

Node.js v18.18.2
```

In a comparable project I've been able to patch it to work with:

```diff
diff --git a/bin/build.js b/bin/build.js
index 346c58657249d0174f4838016f4872704f74bbe0..4285d161cc8ef84d55f473eb873ac9adaa350a61 100644
--- a/bin/build.js
+++ b/bin/build.js
@@ -9,7 +9,7 @@ const dependentHasExifReaderConfig = require('./findDependentConfig');
 process.chdir(path.join(__dirname, '..'));
 
 if (!process.argv.includes('--only-with-config') || checkConfig()) {
-    execSync('webpack', {stdio: 'inherit'});
+    execSync('npx webpack', {stdio: 'inherit'});
 }
 
 function checkConfig() {

```

**Comment by mattiasw** [comment]
Hi! Thanks for reporting. `npx` seems to work fine. You can try the PR out if you want, otherwise I'll merge in a couple of days. #261

**Comment by mattiasw** [comment]
Released as version 4.17.0.
